app/views/shopify_app/sessions/enable_cookies.html.erb

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <link rel="stylesheet" href="https://sdks.shopifycdn.com/polaris/2.5.0/polaris.min.css" />
```

Comment:
This "redirect" page is loading 134KB of styles. Can you please extract the required styles and inline them like you did right here below?